### ISSUE TITLE
Wrong URL generation

### ISSUE THREAD

**Issue by philippneugebauer** [stavro/arc#244]
### Environment

* Elixir version (elixir -v): 1.5.2
* Arc version (mix deps): 0.8.0
* Arc dependencies when applicable (mix deps):
* Operating system: Ubuntu on Windows

### Description

I want to load a file which I stored in S3, but I cannot get the right URL, even with the settings stated in the `README`. The link I get is the following https://s3.amazonaws.com/bucket/images/avatars/4/thumb.png, but I need to get this one:
https://s3.us-east-2.amazonaws.com/bucket/images/avatars/4/thumb.png
config :ex_aws,
  region: "us-east-2",
  s3: [
    scheme: "https://",
    host: "s3.us-east-2.amazonaws.com",
    region: "us-east-2"
  ]

**Comment by pedep** [comment]
It seems you need to follow the directions here : https://github.com/stavro/arc#url-generation under the "asset host" subheading

**Comment by philippneugebauer** [comment]
Yeah, you were right. Following the description and having read that part I thought it doesn't apply to me while the `ex_aws`-config seemed to be the applicable part.

As I have checked the `ex_aws` Readme, their config just lists the region but not the following part:
```
  s3: [
    scheme: "https://",
    host: "s3.us-east-2.amazonaws.com",
    region: "us-east-2"
  ]
```
So to avoid my confusion for others, is this part really required or can it be removed?

**Comment by pedep** [comment]
As far as i can see, the `s3: [...]` part is required to some extent. I think the reason its configured separately, is to allow for completely separate configs for each sub-module of ex_aws, like they do here https://github.com/ex-aws/ex_aws/blob/680a01404ebfbb10a36ec9eeda82777acf60b67d/config/test.exs
